freqtrade/optimize/hyperopt_loss_sharpe_daily.py

```diff
@@ -28,25 +28,33 @@ def hyperopt_loss_function(results: DataFrame, trade_count: int,
 
         Uses Sharpe Ratio calculation.
         """
-        # get profit_percent and apply slippage of 0.1% per trade
-        results.loc[:, 'profit_percent_after_slippage'] = results['profit_percent'] - 0.0005
+        resample_freq = '1D'
+        slippage_per_trade_ratio = 0.0005
+        days_in_year = 365
+        annual_risk_free_rate = 0.03
```

Comment:
This seems like an odd number to pick.
if you assume risk-free as having money in your bank account - then (at least in Europe) - that number would be more like 0.00000000001) ... 🙄

Reply:
I'd just use 0 for risk free rate, it doesn't have an impact in hyperopt context, since we only use sharpe ratio as a baseline measure, rather than for our interest in its final absolute value.

@hroff-1902 Also, you're deducting risk_free_rate from total_profit, while it should be deducted from expected_returns_mean.

Finally, is there any reason why you switched np.std(total_profit) back to total_profit.std() ? I thought we preferred the former.

Reply:
actually, according to the definition of the classic Sharpe Ratio, risk-free rate depends on the market (stock, ETF, etc) and crypto are more volatile... Seen this rate used somewhere as an example, do not remember exactly...